salt/transport/server.py

```diff
@@ -37,8 +37,12 @@ def factory(opts, **kwargs):
             import salt.transport.local
 
             return salt.transport.local.LocalServerChannel(opts)
+        elif ttype == "rabbitmq":
+            import salt.transport.rabbitmq
+
+            return salt.transport.rabbitmq.RabbitMQReqServerChannel(opts)
         else:
-            raise Exception("Channels are only defined for ZeroMQ and TCP")
+            raise Exception("Channels are only defined for ZeroMQ, TCP or RabbitMQ")
```

Comment:
**suggestion** generally, we should clean this error message and make it more helpful, e.g. `raise Exception("Channel {!r} is undefined. Possible channels are zeromq, tcp, or rabbitmq".format(ttype))`

Reply:
Sure.